data/fields/taxi_vehicle.json

```diff
@@ -0,0 +1,13 @@
+{
+    "key": "taxi_vehicle",
+    "type": "combo",
+    "label": "Type",
+    "strings": {
+        "options": {
+            "motorcar": "Standard car",
+            "motorcycle": "Motorcycle, with passengers 'riding pillon' (i.e. mototáxi)",
+            "auto_rickshaw": "Motorized rickshaws (e.g. tuk-tuk, bajaj, etc.)",
+            "cycle_rickshaw": "Pedal-powered rickshaws (e.g. padyak, velotaxi, etc.)"
```

Comment:
I would refrain from listing examples in the option strings. Otherwise they just get too long. There is the possibility to add a separate `description` to the values of `combo` fields, which show up as a hover text in iD (for syntax, see this [example](https://github.com/openstreetmap/id-tagging-schema/blob/v6.3.0/data/fields/flashing_lights.json#L9)).

Reply:
This is a great suggestion. Thanks!